reddit/autorefresh.go

```diff
@@ -0,0 +1,48 @@
+package reddit
+
+import (
+	"log"
+	"time"
+
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2"
+)
+
+type autoRefresh struct {
+	refreshTokenTimer time.Duration
+	cancelAutorefresh context.CancelFunc
+	context           context.Context
+}
+
+func newAutoRefresh() *autoRefresh {
+	ctx := context.Background()
+	ctx, cancel := context.WithCancel(ctx)
```

Comment:
Are these ever used?

Reply:
They are not. I added them as a way to discuss if whether or not we would need to stop the goroutine somewhere.